components/blitz/src/omero/cmd/graphs/GraphHelper.java

```diff
@@ -51,15 +56,34 @@
 
     private final Helper helper;
     private final GraphPathBean graphPathBean;
+    private final LightAdminPrivileges adminPrivileges;
 
     /**
      * Construct a helper for a graph request instance.
      * @param helper the general request helper for the graph request instance
      * @param graphPathBean the graph path bean
+     * @param adminPrivileges the light administrator privileges helper
      */
-    public GraphHelper(Helper helper, GraphPathBean graphPathBean) {
+    public GraphHelper(Helper helper, GraphPathBean graphPathBean, LightAdminPrivileges adminPrivileges) {
         this.helper = helper;
         this.graphPathBean = graphPathBean;
+        this.adminPrivileges = adminPrivileges;
+    }
+
+    /**
+     * Check if the current user is an administrator.
+     * @param requiredPrivilege the privilege that the administrator must have if they are a light administrator
+     * @return if the current user is an administrator
+     */
+    public boolean checkIsAdministrator(AdminPrivilege requiredPrivilege) {
+        final EventContext eventContext = helper.getEventContext();
+        if (!eventContext.isCurrentUserAdmin()) {
```

Comment:
So any privilege makes a user an admin, or for any privilege to be considered valid, the user must be an admin? To some extent, I wonder if the `ec.isCurrentUserAdmin` method doesn't need to be deprecated in favor of this method (refactored up) since the rest of the code-base will be assuming a binary state as opposed to the new spectrum.